apps/ncaasoftscores/manifest.yaml

```diff
@@ -0,0 +1,8 @@
+---
+id: ncaasoft-scores
+name: Softball Scores
+summary: NCAA Softball scores
+desc: Creating multiple app instances is no longer necessary. Just install once and remove any other instances. You can set the rotation speed of scores in the settings.
```

Comment:
Thinking this description might be redundant for a completely new app?

Reply:
Yeah, could be at this point, though there may be some people used to the old app instances. Maybe once the new apps have been live for say a month I can update all descriptions?

Reply:
Sure, WFM. =)